pom.xml

```diff
@@ -63,6 +65,7 @@
         <module>task</module>
         <module>client</module>
         <module>examples</module>
+        <module>commons</module>
```

Comment:
Will do that. Is there any particular convention regarding this which I am not aware of ?